### PR TITLE
Add meta.yaml file for momlevel package

### PR DESCRIPTION
**Pull request by jkrasting**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by jkrasting** [comment]
@conda-forge/help-python, ready for review!

File: recipes/momlevel/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
+    - pip
```
Comment (BastianZim) [inline recipes/momlevel/meta.yaml:20]:
```suggestion
    - pip
    - setuptools-git-versioning
```
Ref: https://github.com/jkrasting/momlevel/blob/1cfa0f4c84172b77535dee87c9bb10c08424e2cb/pyproject.toml#L2

File: recipes/momlevel/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+about:
+  home: https://github.com/jkrasting/momlevel
+  summary: Tools to diagnose sea level in the MOM Ocean Model
+  description: |
+    momlevel is a Python package in the public domain that provides
+    documentation and routines for analyzing sea level and related
+    quantities in MOM6-class models. The package initially focuses
+    on global and local steric sea level change in Boussinesq configurations.
+  license: Public Domain
```
Comment (BastianZim) [inline recipes/momlevel/meta.yaml:47]: This license is not allowed, as mentioned by the linker. If it is not a standard license, please create a custom License Ref: https://conda-forge.org/docs/maintainer/adding_pkgs.html#spdx-identifiers-and-expressions

**Review by BastianZim — CHANGES_REQUESTED** [review]
Just a couple of small things, then this is ready to go 🚀

**Comment by jkrasting** [comment]
Thanks @BastianZim!

@conda-forge/help-python, ready for review!

**Comment by jkrasting** [comment]
@conda-forge-admin, please restart ci

**Pull request merged**